owm_legacy/tests.py

```diff
@@ -1,53 +1,68 @@
-from django.urls import reverse
 from django.test import TestCase
+from django.urls import reverse
 
-from django_netjsonconfig.models import Config, Device
-from django_netjsonconfig.tests import CreateConfigMixin
+from openwisp_controller.config.models import Config, Device
+from openwisp_controller.config.tests import CreateConfigMixin
+from openwisp_users.tests.utils import TestOrganizationMixin
 from owm_legacy.settings import ALLOWED_SUBNETS
 
 
-class TestOwmLegacy(CreateConfigMixin, TestCase):
+class TestOwmLegacy(CreateConfigMixin, TestOrganizationMixin, TestCase):
     """
     tests for owm_legacy
     """
+
     config_model = Config
     device_model = Device
 
     def test_get_config_md5(self):
         c = self._create_config()
-        response = self.client.get(reverse('owm:get_config_md5', args=[c.mac_address]))
-        self.assertEqual(response['Content-Disposition'], 'attachment; filename={0}'.format(self.TEST_MAC_ADDRESS))
+        response = self.client.get(
+            reverse('owm_legacy:get_config_md5', args=[c.mac_address])
+        )
+        self.assertEqual(
+            response['Content-Disposition'],
+            'attachment; filename={0}'.format(self.TEST_MAC_ADDRESS),
+        )
         self.assertEqual(len(response.content), 32)
         checksum1 = response.content
         sleep(1)
-        response = self.client.get(reverse('owm:get_config_md5', args=[c.mac_address]))
+        response = self.client.get(
+            reverse('owm_legacy:get_config_md5', args=[c.mac_address])
+        )
         checksum2 = response.content
         self.assertEqual(checksum1, checksum2)
         c.refresh_from_db()
-        self.assertIsNotNone(c.last_ip)
+        self.assertIsNotNone(c.device.last_ip)
 
     def test_get_config(self):
         d = self._create_device(name='test')
         c = self._create_config(device=d)
-        response = self.client.get(reverse('owm:get_config', args=[c.mac_address]))
-        self.assertEqual(response['Content-Disposition'], 'attachment; filename=test.tar.gz')
+        response = self.client.get(
+            reverse('owm_legacy:get_config', args=[c.mac_address])
+        )
+        self.assertEqual(
+            response['Content-Disposition'], 'attachment; filename=test.tar.gz'
+        )
 
     def test_last_ip(self):
         c = self._create_config()
-        response = self.client.get(reverse('owm:get_config', args=[c.mac_address]))
+        self.client.get(reverse('owm_legacy:get_config', args=[c.mac_address]))
         c.refresh_from_db()
-        self.assertIsNotNone(c.last_ip)
+        self.assertIsNotNone(c.device.last_ip)
 
     def test_status(self):
         c = self._create_config()
-        response = self.client.get(reverse('owm:get_config', args=[c.mac_address]))
+        self.client.get(reverse('owm_legacy:get_config', args=[c.mac_address]))
         c.refresh_from_db()
-        self.assertEqual(c.status, 'running')
+        self.assertEqual(c.status, 'modified')
```

Comment:
`running` was renamed to `applied`, I wonder why now here it changed to `modified`, any idea?